src/components/my-journey/contestDetailTimer.tsx

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
line 4 `ITimeLeft ` why `I` in front of the word TimeLeft?

Reply:
I think it would be better to extract the helper function getRemainingTime from the component and place it in a suitable folder (maybe in utils folder). as It discussed in [my-journey/ContestCardTimer.tsx]